tests/Makefile.am

```diff
@@ -77,6 +77,7 @@ dist_braille_specs_TESTS =				  \
 	braille-specs/bn.yaml				  \
 	braille-specs/chr-us-g1_harness.yaml		  \
 	braille-specs/cop-eg.yaml			  \
+	braille-specs/critical-apparatus.yaml		  \
```

Comment:
Did you mean `braille-specs/hbo.yaml` here?

Reply:
I'll fix that